pkg/utils/kubeclient/node.go

```diff
@@ -24,7 +24,7 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/client"
 )
 
-// Get the latest node info
+//GetNode gets the latest node info
```

Comment:
Please add a space between "//" and "GetNode"

Reply:
Ok, I've completed the modification.